src/main_app/jobs_workers/public_jobs_workers/copy_svg_langs_per_file/steps/__init__.py

```
@@ -0,0 +1,12 @@
+from .extract_text import extract_text_step
+from .extract_titles import extract_titles_step
+from .extract_translations import extract_translations_step
+from .inject_one_file import InjectResult, inject_step_one_file
+
+__all__ = [
+    "extract_translations_step",
+    "inject_step_one_file",
+    "InjectResult",
+    "extract_text_step",
+    "extract_titles_step",
+]
```

Ruff (0.15.17)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)